lib/openstudio/metadata/serializer.rb

```diff
@@ -50,10 +60,16 @@ def initialize(building_namespace: 'http://example.com/mybuilding#')
           brick: @brick,
           bldg: @bldg
         }
-        @g = RDF::Repository.new
```

Comment:
I like to try and keep attribute definitions in the `init` of a class, even if they are blank.  So, I think either having `@g = nil` and reassigning it in the `create_graph_from_entities` method, or just instantiating the new repository here.  Work?

Reply:
yeah that sounds fine, I'll add the `@g = nil` back in the file